src/hooks.ts

```diff
@@ -0,0 +1,6 @@
+export const redeploy = async () => {
+  await fetch(
+    "https://api.cloudflare.com/client/v4/pages/webhooks/deploy_hooks/5d30d67d-725a-4c5f-9b60-83f1a457ac9d",
```

Comment:
This url should probably not be in the public repo. Use a environment variable instead.

Reply:
I have added the (new) deploy hook to coolify:
https://coolify.youngvision.work/project/t04c44g/production/application/v8kgggw#environment-variables